src/TLuaInterpreter.cpp

```diff
@@ -17153,10 +17153,12 @@ int TLuaInterpreter::showNotification(lua_State* L)
     if (n >= 2) {
         text = getVerifiedString(L, __func__, 2, "message");
     }
-    int notificationExpirationTime = 1000;
+    auto notificationExpirationTime = 1s;
```

Comment:
I can't see how `auto` is appropriate here - the argument the **[`QSystemTrayIcon::showMessage`](https://doc.qt.io/qt-5/qsystemtrayicon.html#showMessage)** method takes an `int` number of mili-seconds and who knows (well *I* don't) what the type used here will be... :man_shrugging: